cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -845,27 +846,29 @@ func validatePerformanceProfile(performanceProfile *performancev2.PerformancePro
 		return false, nil
 	}
 
-	if *performanceProfile.Spec.HugePages.DefaultHugePagesSize != "1G" {
-		return false, nil
-	}
-
 	if len(performanceProfile.Spec.HugePages.Pages) == 0 {
 		return false, nil
 	}
 
-	if performanceProfile.Spec.HugePages.Pages[0].Count < 10 {
-		return false, nil
-	}
+	find1GHugePages := false
+	for idx := range performanceProfile.Spec.HugePages.Pages {
```

Comment:
why use the index notation and not the item one here?

Reply:
done